README.md

```diff
@@ -67,8 +88,8 @@ fetch(id: string): RemoteCollection<Resource>
 ### Examples
 ```ts
 const users: User[] = [{ id: 'a', name: 'Alice' }, { id: 'b', name: 'Bob' }];
-const collection = new RemoteCollection<User>()
-  .withList('id', users)
+const collection = new RemoteCollection<User>('id')
```

Comment:
@scotttrinh a common use case i've had / might be cool to document, is setting a whole list of resources to pending/refresh; e.g. ['a','b','c'...]

Reply:
kinda like `concat` but for states other than initial addition

Reply:
@dylanpyle 

If it's at a view, that is what `refresh` does:

```ts
const collection = new RemoteCollection<User>('id')
  .withList(users, 'team1')
  .refresh('team1')
  .refresh('team2');

assert.deepStrictEqual(
  collection.view('team1'),
  RemoteData.refresh(users)
);

assert.deepStrictEqual(
  collection.view('team2'),
  RemoteData.pending
);
```

— Is that roughly what you were looking for? `.concat` adds, so I'm not sure if that covers the use case you had in mind

Reply:
Here's a helpful cheatsheet:

| RemoteState | collection | resource |
|---|---|---|
|`RemoteInitial`|`reset`|`remove`|
|`RemotePending`|`refresh`|`fetch`|
|`RemoteFailure`|`withListFailure`|`withResourceFailure`|
|`RemoteRefresh`|`refresh`|`fetch`|
|`RemoteSuccess`|`withList`|`withResource`|

I'll try to find some place to put this in the docs.

Reply:
per slack discussion: i can’t find a good use case for this in our code, i think i’m thinking of something else. all good.